.github/workflows/build-and-check-fleetctl-docker-and-deps.yml

```diff
@@ -88,15 +99,23 @@ jobs:
           vuln-type: "os,library"
           severity: "CRITICAL"
 
-      - name: Run Trivy vulnerability scanner on fleetdm/fleetctl
-        uses: aquasecurity/trivy-action@6e7b7d1fd3e4fef0c5fa8cce1229c54b2c9bd0d8
-        env:
-          TRIVY_DB_REPOSITORY: public.ecr.aws/aquasecurity/trivy-db
-          TRIVY_JAVA_DB_REPOSITORY: public.ecr.aws/aquasecurity/trivy-java-db
+      - name: Slack root notification
+        if: ${{ steps.check_root.outputs.root_warn  == 'true' }}
```

Comment:
Where is this "check_root" step?

Reply:
Busted! (Copy/paste from other actions.)